product-management/index.md

```diff
@@ -0,0 +1,83 @@
+
+Once the solution and KPI's are agreed upon, update the PRD document.
+
+#### 3. Product Development & Execution
+
+**Goal: Work with engineering to plan and execute.**
+
+Once the problem, solution and KPI's are documented, you should share the document with the tech lead and/or engineers and ask him/her to design the solution and help break down the work into small techncial tasks with rough estimates. You should create [JIRA](https://www.atlassian.com/software/jira) tickets based on those work items. For planning purpose, we use agile methodology. Our sprints are 2 weeks long and we recommend product manager to conduct following scrum ceremonies:
+
+-   **Sprint planning (Duration: 60 minutes, When: 2 days before sprint starts)**: In the sprint planning, you present the work items that you would like engineering team to achieve. This is an opportunity for engineering team to ask questions and for you to get their buyoff on the plan.
+
+-   **Daily standups (Duration: 15 minutes, When: Daily)**: This is daily touch base call with the product team (engineering, design). During the meeting, go through the status update and ensure you understand the progress and mkae sure that the overall sprint remains on track.
+
+-   **Sprint Exit (Duration: 60 minutes, When: Last day of sprint)**: In the morning of the last day of the sprint, send engineering team a reminder to ensure that they should focus on preparing for the demo.
+
+-   **Sprint Retro (Duration: 30 minutes, When: 2 days after sprint exit)**: The goal of this meeting is to reflect what went well and areas to improve for the sprint with clear action items.
+
+#### 4. Measure, Learn & Improve
+
+**Goal: Observe feature usage, learn, and improve.**
+
+Based on the KPI's decided during evangelizing the solution, you need to start monitoring the product usage once the feature is launched. Often, in early stages of the feature launch, you may not get sufficient data to make meaningful conclusion. This is where you need to mix data with your intuition. Based on data and your intuition, you should decide whether the feature launch was successful or not and refine the solution accordingly.
+
+At JTC, we recommend using following tools:
+
+-   [MixPanel](https://mixpanel.com/) to capture usage data and building KPI reports.
+-   [Inspectlet](https://www.inspectlet.com/) to capture user sessions as video to review them. We expect you to review the few sessions manually every day to see how the feature is being used or where users are often stuck.
+
+### Onboarding
+
+#### Week 1: Understand the product and take charge of scrum
+
+##### Day 1:
+
+-   Setup 30 minute 1:1 with tech lead. Few things to talk about - (1) Get product overview and product access and relevant tools (2) Get invited to all sprint ceremonies and become owner of those meetings (3) What's working well and what's not working well
```

Comment:
First step even before Tech Lead - 

Setup a 30-minute onboarding and welcome call with Mohit to go through company processes and the product you will be driving.